doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -20,3 +20,43 @@ bin/console pimcore:documents:cleanup printpage printcontainer
 DELETE FROM `users_permission_definitions` WHERE `key` = 'web2print_settings';
 UPDATE `users` SET `permissions`=REGEXP_REPLACE(`permissions`, '(?:^|,)web2print_settings(?:$|,)', '') WHERE `permissions` REGEXP '(?:^|,)web2print_settings(?:$|,)';
 ```
+
+### Storage Config
+If you use `symfony-config` as target in your `config_location` replace the following nodes in the config files for the following bundles: Custom Reports, Web2Print, Static Routes
```

Comment:
Actually there's no `config_location` in v10, right? So we need to explain the migration from env-variables to the symfony config.